eessi_bot_job_manager.py

```diff
@@ -85,6 +83,8 @@ def get_current_jobs(self):
                     "state": job[4],
                     "reason": job[8],
                 }
+                if current_jobs[job[0]]['state'] in bad_state_codes:
+                    error("Job {} in state {}: {}".format(job[0], job[4], bad_state_messages[job[4]]))
```

Comment:
I'm not sure if we should make the job manager *crash* if it hits a job that has timed out, for example...
Instead, the job manager should report that back to the PR, and continue, ideally?

Code-style wise: we're accessing the "`state`" value in multiple ways here, that could be done better.
Also, we should report an error in the `else` case, i.e. if don't get exactly 9 "fields"?

Something like:

```python
if len(job) == 9:
    job_id, job_state, job_state_reason = job[0], job[4], job[8]
    current_jobs[job_id] = {
        "jobid": job_id,
        "state": job_state,
        "reason": job_reason,
    }
    if job_state in bad_state_codes:
        error("Job {} in state {}: {}".format(job_id, job_state, bad_state_messages[job_state]))
else:
    error("Incorrect job info, expected 9 fields, found {}: {}".format(len(job), job))

Reply:
Is there any possibility that there won't be 9 fields? I thought Slurm output always had the same fields.

Reply:
Hmm, the number of output fields could be depending on the configuration or build of Slurm.